Model/Contexts/PlaylistDataContext.cs

```diff
@@ -1,27 +1,48 @@
 ﻿using System;
 using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
 using System.Text;
+using Model.Annotations;
+using Model.DbModels;
 
 namespace Model
 {
-    public class PlaylistDataContext
+    public class PlaylistDataContext : INotifyPropertyChanged
     {
         public string PlaylistName { get; set; }
         public string Description { get; set; }
         public List<SongInfo> PlaylistItems { get; set; }
+        public Playlist Playlist { get; set; }
+        public List<Song> Songs { get; set; }
 
-        public PlaylistDataContext()
+        public void AddPlaylistsToMenu()
         {
-            PlaylistName = "Test playlist";
-            Description = "a description for the test playlist";
+            PlaylistName = Playlist.Name;
+            Description = Playlist.Description;
 
-            PlaylistItems = new List<SongInfo>
+            PlaylistItems = new List<SongInfo>();
+
+            if (Playlist.PlaylistSongs != null && Playlist.PlaylistSongs.Count != 0)
             {
-                new SongInfo("test nummer 1", "een artiest", "06:66", "today"),
-                new SongInfo("test nummer 2", "nog een artiest", "04:20", "today"),
-                new SongInfo("test nummer 3", "een andere artiest", "00:00", "today"),
-                new SongInfo("test nummer 4", "dezelfde artiest", "12:34", "today"),
-            };
+                foreach (var song in Playlist.PlaylistSongs)
```

Comment:
Kun je dit ook ombouwen naar een Foreach? Dat is namelijk sneller :)

Reply:
dit is al een foreach, toch?

Reply:
of bedoel je iets als Playlist.PlaylistSongs.ForEach(Action<PlaylistSong> action)? dat kan wel, maar dan moet die lijst eerst gecast worden naar een List. het is nu namelijk een IList. Ik weet niet of dat dan nog steeds sneller is, maar zo ja kan ik het zo omgooien